hypha/apply/projects/templates/application_projects/project_admin_detail.html

```diff
@@ -48,26 +48,24 @@ <h4 class="modal__header-bar">{% trans "Approve Contract" %}</h4>
 </div>
 {% endif %}
 
-{% if object.requires_approval %}
-    <a
-        {% if not object.can_send_for_approval %}
-            onclick="return false"
-            data-tooltip="
-                {% if not object.lead %}
-                    {% trans "A lead must be assigned" %}
-                {% elif not object.user_has_updated_details %}
-                    {% trans "Project approval form must be completed" %}
-                {% elif object.can_make_approval or object.can_make_final_approval %}
-                    {% trans "Currently awaiting approval" %}
-                {% endif %}"
-        {% endif %}
-        data-fancybox
-        data-src="#send-for-approval"
-        class="button button--bottom-space button--primary button--full-width {% if not object.can_send_for_approval or not user.is_apply_staff %}button--tooltip-disabled{% endif %}"
-        href="#">
-        {% trans "Submit for Approval" %}
-    </a>
-{% endif %}
+<a
+    {% if not object.can_send_for_approval %}
+        onclick="return false"
+        data-tooltip="
+            {% if not object.lead %}
+                {% trans "A lead must be assigned" %}
+            {% elif not object.user_has_updated_details %}
+                {% trans "Project approval form must be completed" %}
+            {% elif object.can_make_approval or object.can_make_final_approval %}
+                {% trans "Currently awaiting approval" %}
+            {% endif %}"
```

Comment:
Too many logic and permission-related things happening here. Consider moving to a python function that returns 1) `can_send_for_appoval` 2) `reason`

The python function can also be quickly unit tested.